### ISSUE TITLE
Adding examples of how to test the form with jest

### ISSUE THREAD

**Issue by keyserfaty** [davidkpiano/react-redux-form#1200]
### The Problem
It would be easier to know how to test for form with jest if there were examples. 

I spent some time last week trying to set everything up on my own codebase so I'll be happy to add testing to the examples in the repo if you are interested.

**Comment by petewins** [comment]
yes, please. I've having trouble mocking the values for my forms without including the entire form state
